build.gradle

```diff
@@ -360,6 +360,15 @@ android {
       dimension = 'brand'
       applicationId = 'org.medicmobile.webapp.mobile.chis_ne'
     }
+    cht_rci {
+      dimension = 'brand'
+      applicationId = 'org.medicmobile.webapp.mobile.cht_rci'
+    }
+    
+    cht_rci_test {
```

Comment:
Do we need to build the test version as well?
We have the PR for this already as #240.
There I had [suggested](https://github.com/medic/cht-android/pull/240#discussion_r769852716) that we can create APK for test locally.

Reply:
Ok, removed!